### PR TITLE
unbound 1.11.0

### PR DESCRIPTION
**Pull request by miccal**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by miccal** [comment]
The only failure for 10.14 and 10.13 was for `agda`, [which is to be expected](https://github.com/Homebrew/homebrew-core/issues/55253).

The tests for 10.15 appear to have stalled.

**Comment by Rylan12** [comment]
The `agda` issue should be fine as it doesn't build from source anyway at the moment.

We do need the 10.15 checks to run, though, in order to get updated bottles. The 10.15 runners have been having some trouble recently for some reason, but rerunning them usually does it. I'll retrigger it. If it doesn't work this time, try rebasing and force-pushing.

**Comment by miccal** [comment]
Thank you @Rylan12, as always.

**Comment by miccal** [comment]
On 10.15 and 10.13:
```
Error: 2 failed steps!
brew test --verbose agda
brew test --verbose csound
```
while on 10.14 only `adga` failed.

[`csound` is a known random failure](https://github.com/Homebrew/homebrew-core/pull/54364#issuecomment-626316365)

So ok to merge I think.

Thanks.

**Comment by miccal** [comment]
> Is there an upstream issue for this?

@SMillerDev I believe it is because of the [Cabal v1 install deprecation](https://github.com/Homebrew/homebrew-core/issues/55253), and is therefore not an upstream issue?

**Comment by SMillerDev** [comment]
In that case someone should make a pull request to fix it.

**Comment by Rylan12** [comment]
It sounds like @Bo98 is working on the `agda` issue. I'm not sure if it's: 1. going well or 2. will be done soon

@SMillerDev I understand not wanting to merge things that have had CI failures, but I think holding this up isn't doing anything positive. `agda` won't build regardless of this PR and I don't think merging this will make fixing `agda` it any easier/harder. Not merging only means we are holding back a new version of `unbound` for users (plus the ~171 formulae that depend on it in some way)

Are there other consequences I'm missing?

**Comment by SMillerDev** [comment]
I'm sure it won't hurt the progress there. But keeping this for another time like this isn't helping anything either. I think we need to be stricter in this and as a policy either have a pull request or an issue open. Currently we keep on rediscovering issues and it's hurting progress like here.

TL;DR we need a PR or issue for `agda` if we're not going to fix it here.

**Comment by Rylan12** [comment]
I think that's a reasonable requirements to stick to. In this case, though, there is an issue open (#55253).

**Comment by SMillerDev** [comment]
https://github.com/Homebrew/homebrew-core/issues/55253 isn't really an issue about `agda` being broken though. That one is meant to guide the migration.

**Comment by Rylan12** [comment]
But the reason `agda` is broken is that cabal v1 is deprecated, so I think it's implied that all formulae in that issue will be broken.

Maybe that's not the case. If it's not, then there definitely should be an issue or PR for it.

**Comment by Bo98** [comment]
> 1. going well

no

> 2. will be done soon

A quick fix is to make it use "v1-install" directly (and no sandbox).

"v2-install" requires either the MacPorts-style hack or wrapper scripts for every binary, both of which should have some `brew` helpers to accomodate.

A hybrid approach of v2-installing dependencies and not using Cabal for the actual `agda` build is another possibility I'm investigating.

I planned to discuss Cabal a bit at the next maintainer call, but I should probably not wait until then and figure something out this week. I should have dealt with it weeks ago since it's my fault really that the test fails.

**Comment by miccal** [comment]
Agda issue reported in https://github.com/Homebrew/homebrew-core/issues/58984

**Comment by miccal** [comment]
@Rylan12, @SMillerDev and @Bo98 I rebased and the only failure now is `agda`.

Thanks.

**Review by Rylan12 — APPROVED** [review]

**Comment by Rylan12** [comment]
Thanks, @miccal!
